src/components/menu/menu.ts

```diff
@@ -49,19 +48,21 @@ export default class SlMenu extends ShoelaceElement {
     if (event.key === 'Enter' || event.key === ' ') {
       const item = this.getCurrentItem();
       event.preventDefault();
+      event.stopPropagation();
```

Comment:
I'm kind of curious why this `stopPropagation()` was adding? Were the events bubbling causing the parent menus to catch the event?